index.html

```diff
@@ -12,206 +12,257 @@
+      </nav>
+
+    </header>
+    <!--Welcome section-->
+
+
+    <main class="main-section">
+
+      <div class="main-title">
+        <h1>I'm Gilbert, Glad to See You!</h1>
+
+        <p>
+          I'm a software developer! I can help you build a product , feature or website Look through some of my work and
+          experience! If you like what you see and have a project you need coded, don't hestiate to contact me.
+        </p>
+
+        <div class="lets-connect"> LET'S CONNECT</div>
+        <!--End of Welcome section-->
+
+        <!-- Social Media  Welcome section-->
+        <div class="icons-container">
+          <ul class="social-media-icons">
+            <li>
+              <a href="#">
+                <img src="media/twitter.svg" alt="twitter" />
+
+              </a>
+            </li>
+            <li>
+              <a href="#">
+                <img src="media/Linkedin-icon.svg" alt="linkedin" />
+              </a>
+            </li>
+            <li>
+              <a href="#">
+                <img src="media/medium.svg" alt="medium" />
+              </a>
+            </li>
+            <li>
+              <a href="#">
+                <img src="media/github.svg" alt="github" />
+              </a>
+            </li>
+            <li>
+              <a href="#">
+                <img src="media/angelist.svg" alt="angellist" />
+              </a>
+            </li>
+          </ul>
+        </div>
       </div>
 
       <!--End of Social media Welcome section-->
-    </section>
-  </main>
+    </main>
+  </section>
 
   <!--These are my projects-->
 
-  <section class="Work-projects">
+  <section class="cards">
 
-    <div class="card">
-      <div class="projects-done">
-        <img src="./media/Snapshoot Portfolio.png" alt="Website" width="98%">
-        <h2> Tonic</h2>
+    <div class="cards-section">
 
-        <ul>
-          <li>CANOPY</li>
-          <li>Back End Dev</li>
-          <li>2015</li>
-        </ul>
+      <div class="two">
+        <img src="./media/Snapshoot Portfolio-d1.png" alt="Website" width="100%">
       </div>
 
-      <div class="container">
-        <p> A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
-      </div>
+      <div class="card-contents two">
+        <div>
+          <h2> Tonic</h2>
+        </div>
+
+        <div>
+          <ul>
+            <li>CANOPY</li>
+            <li>Back End Dev</li>
+            <li>2015</li>
+          </ul>
+        </div>
+
 
-      <div>
-        <ul class="skills">
-          <li class="skill-html">
-            <a href="#" class="skill-content">html</a>
-          </li>
-          <li class="skill-css">
-            <a href="#" class="skill-content">css</a>
-          </li>
-          <li class="skill-js">
-            <a href="#" class="skill-content">javascript</a>
-          </li>
-        </ul>
-      </div>
 
-      <div class="button">
-        <p class="on-p">See Projects</p>
+        <div>
```

Comment:
- [ ] Please change this `p` tag with an `a` or button `tag`. because it lets you to connect with other parts of the sections. and please add the button interactions like hover, enable. when i click or hover it does not work. please work on it.
## your design👇
 
![btnst](https://user-images.githubusercontent.com/66992901/194419125-6b00bfd3-9484-4214-84eb-86feb721379d.PNG)

## figma design👇
![button](https://user-images.githubusercontent.com/66992901/194419097-363dd813-1d2b-4975-b43f-df8852f340d8.PNG)
#### please do follow with all your buttons in your project.